demo/demo-springmvc/springmvc-server/src/main/java/org/apache/servicecomb/demo/springmvc/server/SpringMvcDefaultValues.java

```diff
@@ -64,4 +64,28 @@ public String query3(@RequestParam("a") @Min(value = 20) int a, @RequestParam("b
     return "Hello " + a + b;
   }
 
+  @PostMapping("/javaprimitive1")
+  public String springJavaPrim1(@RequestParam(name = "a") int a,
```

Comment:
From this demo and I made some tests, It seems for SpringMVC, there is a bug. 
For @RequestParam and other annotations, they got a required attribute, and default value is true. It seems that when generating swagger, we do not consider this attribute and it is always false. Event we write @RequestParam(name="a", requrired=true). Can you check that?

Reply:
yes,you are correct when we made @RequestParam(name="a", requrired=true) then also in swagger file it is showing required field is false. Looks to be defect in swagger generation module about springmvc.

Reply:
I fixed the base springmvc defect related to required parameter and raised PR. please check it.

Reply:
This default required is true, seems this test case will fail

Reply:
yes...i mofied in local evn but forgot to merge the demo. i am raising PR now. Thank you !